util/cpuprofile/consumer.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021 PingCAP, Inc.
```

Comment:
`consumer.go` is too general. I think this file can be combined with `handler.go` as `pprof_api.go`